_data/simple-icons.json

```diff
@@ -12499,6 +12499,11 @@
 			"hex": "070E10",
 			"source": "https://www.prefect.io"
 		},
+		{
+			"title": "PreMiD",
+			"hex": "7289DA",
+			"source": "https://github.com/PreMiD/PreMiD/blob/f9a976ef1dfab92fac87b008b7338a802c97567c/src/assets/tray/icon.svg"
```

Comment:
```suggestion
			"source": "https://cdn.rcd.gg/PreMiD-icon.svg"
```

Reply:
I can't open the URL. It takes a very long time to long and no response.

Could you help to check?

Reply:
> I can't open the URL. It takes a very long time to long and no response.
> 
> Could you help to check?

Should work now.